recipes/ostaphep/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "ostapHEP" %}
+{% set version = "1.3.2.0" %}
+{% set hash = "9ce0c36911c5286317d16b04bf1c0b942a1cce74c6462e47fcfcce764f6579eb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OstapHEP/ostap/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
+  folder: ostap
+
+build:
+  number: 0
+  skip: True  # [not linux]  
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - {{ compiler('c') }}  
+    - {{ compiler('cxx') }}  
+    - ROOT >=6.16.0
+    - python 
+    - ipython
+  run:
+    - compilers
+    - python 
+    - ROOT 
+    - ipython
+test:
+  requires:
```

Comment:
These requirements shouldn’t be needed as they should be always be installed (as they’re runtime requirements).

This can be simplified to:
```yaml
test:
  imports:
    - ostap
    - ostap.core.core.Ostap
```